superset/views/core.py

```diff
@@ -2268,6 +2269,11 @@ def sqllab(self):
             'superset/sqllab.html',
             bootstrap_data=json.dumps(d, default=utils.json_iso_dttm_ser)
         )
+
+    @has_access
+    @expose("/rest/api/getLocale", methods=['GET', 'POST'])
+    def getLocale(self):
+        return str(get_locale())
```

Comment:
Because the English language that the 'API' returns is' en ', but the English translation file's directory is' es'.